.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,17 @@
+name: Documentation improvement
+description: Create a report to help us improve the documentation. Alternatively you can just open a pull request with the suggested change.
+labels: [Documentation]
```

Our "documentation" label starts with a lower-case d
```suggestion
labels: [documentation]
```